pkg/controllers/state/statenode.go

```diff
@@ -108,17 +110,27 @@ func NewNode() *StateNode {
 	}
 }
 
+func (in *StateNode) OwnerKey() nodepoolutil.Key {
```

Comment:
Does it make sense for a NodeClaim to have two of these labels? Would that break the logic at all? Seems like it wouldn't, and the NodePool Label would just be ignored, which if we can, we should make it the other way around.

Reply:
Good point. I'll flip the ordering here.